package.json

```diff
@@ -32,6 +32,7 @@
     "detective": "^4.2.0",
     "detective-es6": "1.1.2",
     "graphviz": "0.0.8",
+    "precinct": "^2.5.1",
```

Comment:
Prefer upgrading detective-es6@1.1.3. That's all we really want from precinct anyway (we don't need the css or amd/cjs detectives to be included).